js/main.ts

```diff
@@ -73,13 +74,14 @@ export default function denoMain() {
   const cwd = startResMsg.cwd();
   log("cwd", cwd);
 
-  const argv: string[] = [];
-  for (let i = 0; i < startResMsg.argvLength(); i++) {
+  // TODO handle shebang.
```

Comment:
Yeah that's interesting. What if the author set `#!/usr/bin/env deno --alllow-write` as the shebang...

Reply:
Same that happens in python:
```
~/src/deno> cat x.py
#!/usr/bin/env python -d
import sys
print sys.argv
~/src/deno> ./x.py
['./x.py']
~/src/deno>
```

`--allow-write` is a privileged parameter, so it should be removed from argv.

Reply:
I meant something else. A script could add --allow-write it it's shebang to get write permissions without user interaction.
But of course the user would still have to chmod +x it.